docs/getting-started/integrate-auth/nextjs.mdx

```diff
@@ -76,12 +174,43 @@ npm run dev
 Then open [http://localhost:3000](http://localhost:3000) in your browser. are presented with Ory's Sign In page. Let's click on
 sign up and create your first user.
 
+  </TabItem>
+
+</Tabs>
+
 ## Go to production
 
+When upgrading to production, you need to set up a custom domain and configure your Next.js app to use it. Once you have
+configured a custom domain and deployed your application the integrations package is no longer needed:
+
+````diff
+- import { edgeConfig } from "@ory/integrations/next"
+- const ory = new FrontendApi(new Configuration(edgeConfig))
++ const basePath = process.env.NEXT_PUBLIC_ORY_SDK_URL;
+
++ const ory = new FrontendApi(
++   new Configuration({
++     basePath: basePath,
++     baseOptions: {
++       withCredentials: true,
++     },
++   })
++ );
+
+...
+
+- return router.push(edgeConfig.basePath + "/ui/login")
++ return router.push(basePath + "/ui/login");
+
+...
+```
+
+You can now also remove the `pages/api/.ory/` route.
+
 Release your app bundled with Ory to production using:
 
 ```mdx-code-block
 import VercelDeploy from '@site/src/components/VercelDeploy'
 
 <VercelDeploy />
-```
+````
```

Comment:
Something's not right here 
![image](https://user-images.githubusercontent.com/18033717/227446419-8cc855fb-b088-494f-a62b-ee1338c7c2ff.png)